### ISSUE TITLE
Plugins is not a Constructor 

### ISSUE THREAD

**Issue by Ifletcher668** [twilio/twilio-cli#185]
Using Ubuntu 18.4

When attempting to run twilio phone-numbers:update, I get an error saying "plugins is not a constructor" 

```
 » Installing ngrok ...
[DEBUG] Found command "phone-numbers:update" plugin: twilio-cli
 » twilio-cli encountered an unexpected error. To report this issue, execute the command with the "-l debug" flag, then copy the output to a new issue here: "https://github.com/twilio/twilio-cli/issues"
[DEBUG] Plugins is not a constructor
[DEBUG] TypeError: Plugins is not a constructor
    at requireInstall (/usr/lib/node_modules/twilio-cli/node_modules/@twilio/cli-core/src/services/require-install.js:104:19)
    at NumberUpdate.install (/usr/lib/node_modules/twilio-cli/node_modules/@twilio/cli-core/src/base-commands/base-command.js:153:12)
    at NumberUpdate.run (/usr/lib/node_modules/twilio-cli/src/commands/phone-numbers/update.js:37:33)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async NumberUpdate._run (/usr/lib/node_modules/twilio-cli/node_modules/@oclif/command/lib/command.js:43:20)
    at async Config.runCommand (/usr/lib/node_modules/twilio-cli/node_modules/@oclif/config/lib/config.js:172:9)
    at async Main.run (/usr/lib/node_modules/twilio-cli/node_modules/@oclif/command/lib/main.js:22:9)
    at async Main._run (/usr/lib/node_modules/twilio-cli/node_modules/@oclif/command/lib/command.js:43:20)

```

**Comment by childish-sambino** [comment]
This occurs in `2.0.2` and was fixed in `2.0.3`.

**Comment by johnmmason** [comment]
I am experiencing the same issue with `2.1.0` on OS X, see below:

```
[DEBUG] Found ngrok version in "twilio-cli" optionalDependencies: ^3.2.7
 » Installing ngrok ...
[DEBUG] Found command "phone-numbers:update" plugin: twilio-cli
 » twilio-cli encountered an unexpected error. To report this issue, execute the command with the "-l debug" flag, then copy the output to a new issue here: "https://github.com/twilio/twilio-cli/issues"
[DEBUG] Plugins is not a constructor
[DEBUG] TypeError: Plugins is not a constructor
    at requireInstall (/usr/local/Cellar/twilio/2.1.0/libexec/node_modules/@twilio/cli-core/src/services/require-install.js:104:19)
    at NumberUpdate.install (/usr/local/Cellar/twilio/2.1.0/libexec/node_modules/@twilio/cli-core/src/base-commands/base-command.js:153:12)
    at NumberUpdate.run (/usr/local/Cellar/twilio/2.1.0/libexec/src/commands/phone-numbers/update.js:37:33)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async NumberUpdate._run (/usr/local/Cellar/twilio/2.1.0/libexec/node_modules/@oclif/command/lib/command.js:43:20)
    at async Config.runCommand (/usr/local/Cellar/twilio/2.1.0/libexec/node_modules/@oclif/config/lib/config.js:172:9)
    at async Main.run (/usr/local/Cellar/twilio/2.1.0/libexec/node_modules/@oclif/command/lib/main.js:22:9)
    at async Main._run (/usr/local/Cellar/twilio/2.1.0/libexec/node_modules/@oclif/command/lib/command.js:43:20)
```

and for reference:

```
% twilio --version
twilio-cli/2.1.0 darwin-x64 node-v14.3.0
```
